### PR TITLE
Added basic support for custom parameters in the docker run command.

### PR DESCRIPTION
**Pull request by marcusrognes**
One possible implementation of what is discussed here: #75

### CODE REVIEW

**Comment by madushan1000** [comment]
@mnmtanish take a look at this

**Comment by marcusrognes** [comment]
@madushan1000 @mnmtanish I can rework this pull request to make the syntax be like:

```
meteor: {
  name: 'app',
  path: '../app',
  servers: {
    one: {}, two: {}
  },
  env: {
    ROOT_URL: 'app.com',
    MONGO_URL: 'mongodb://localhost/meteor'
  },
  docker: {
    image: 'madushan1000/meteord-test',
    args: [
      '--log-driver syslog',
      '--log-opt syslog-address="udp://syslogserverurl.com:1234"',
    ],
  },
},
```

I think it would be better to put the `dockerImage` parameter there to, but that could break backwards compatibility if we removed the original option.

**Comment by MasterJames** [comment]
Break it +1 easy to fix upon updating

**Comment by madushan1000** [comment]
Yeah, It looks better!

**Comment by marcusrognes** [comment]
@madushan1000 @mnmtanish I have updated to the new syntax now.

File: src/modules/meteor/index.js [hunk 1]
```diff
@@ -104,7 +104,10 @@ export function push(api) {
           sslConfig: config.ssl,
           logConfig: config.log,
           volumes: config.volumes,
-          image: config.dockerImage || 'meteorhacks/meteord:base'
+          docker: Object.assign({
+            image:'meteorhacks/meteord:base',
```
Comment (thani-sh) [inline src/modules/meteor/index.js]: Hi @marcusrognes 
With a small tweak we can keep mup compatible with older config files. Can you check whether `config.dockerImage` exists and use it. If we do it in the `mup-api.js` file [here](https://github.com/marcusrognes/meteor-up/blob/5bba2e14ed888436ef659a589c9df076d1e9319d/src/mup-api.js#L18) we don't have to worry about it anywhere else.

``` js
// NOTE: backward compatibility
if (!this.config.docker && this.config.dockerImage) {
  this.config.docker = {image: config.dockerImage};
  delete this.config.dockerImage;
}
```

Thanks

**Comment by marcusrognes** [comment]
How about this?

```
docker: Object.assign({
  image:config.dockerImage || 'meteorhacks/meteord:base',
  args:[]
},config.docker)
```

or is there a good reason to delete the `dockerImage` parameter afterwords?

@mnmtanish

**Comment by arunoda** [comment]
@mnmtanish @madushan1000 can we take this?

**Comment by thani-sh** [comment]
LGTM 👍

**Comment by krishaamer** [comment]
How's it going?

**Comment by madushan1000** [comment]
I refactored the code from here and merged to the master in #224

**Pull request closed without merging**